### PR TITLE
OEL-1453: Consolidate theme regions

### PR DESCRIPTION
**Pull request by escuriola**
### Description

Configure oe_whitelabel navigation blocks on oe_showcase

### Change log

- Added:oe_showcase_post_update
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: oe_showcase.post_update.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ */
+
+declare(strict_types=1);
+
+use Drupal\block\Entity\Block;
+
+/**
+ * Set the updated regions of the oe_whitelabel navigation blocks.
+ */
+function oe_showcase_post_update_10001(&$sandbox) {
```
Comment (brummbar) [inline oe_showcase.post_update.php]: Let's start from 00001.

File: oe_showcase.post_update.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    'oe_whitelabel_local_tasks' => 'highlighted',
+    'oe_whitelabel_messages' => 'highlighted',
+    'oe_whitelabel_main_navigation' => 'navigation_left',
+    'oe_whitelabel_search_form' => 'navigation_right',
+  ];
+
+  foreach ($blocks as $block_id => $region) {
+    $block = Block::load($block_id);
+
+    if ($block->getTheme() === 'oe_whitelabel') {
```
Comment (brummbar) [inline oe_showcase.post_update.php]: Is there a specific reason for checking the theme? That block ID is specific to our theme, plus in showcase we can assume the state of the config.

**Review by brummbar — CHANGES_REQUESTED** [review]
Two minor remarks.

**Review by brummbar — DISMISSED** [review]

**Review by drishu — CHANGES_REQUESTED** [review]
You need to update the region of oe_whitelabel_search_form also in oe_showcase_search

**Review by drishu — APPROVED** [review]

**Review by drishu — DISMISSED** [review]

**Review by brummbar — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**